### ISSUE TITLE
Files not as attachments

### ISSUE THREAD

**Issue by JensenR** [terminal42/contao-fineuploader#108]
In Contao 4.13, PDF files were attached in the sent email. In Contao 5.3 the links always appear in the email but no email attachment. I compared the settings in a 4.13 and 5.3 version and set them exactly the same. Unfortunately without success. Is this a bug?

**Comment by fritzmg** [comment]
This is actually a regression in Contao 5: https://github.com/contao/contao/pull/7039

**Comment by fritzmg** [comment]
Nevertheless, I have implemented a fix in `3.4.10`.

**Comment by hendrikbengs** [comment]
After upgrading to 3.4.10 I get the following error message (Contao 4.13.39, PHP 8.2) when I upload a PDF file.

`The "%s" file does not exist or is not readable.`

Excerpt from the Stack Trace:

```
Symfony\Component\Mime\Exception\InvalidArgumentException:
The "###" file does not exist or is not readable.

  at vendor/symfony/mime/FileinfoMimeTypeGuesser.php:50
  at Symfony\Component\Mime\FileinfoMimeTypeGuesser->guessMimeType('###')
     (vendor/symfony/mime/MimeTypes.php:134)
  at Symfony\Component\Mime\MimeTypes->guessMimeType('###')
     (vendor/terminal42/notification_center/src/Util/FileUploadNormalizer.php:66)
…
```
